lib/monitoring/logging-engine.js

```diff
@@ -0,0 +1,145 @@
+const LogLevel = {
+  OFF: 0,
+  TRACE: 1,
+  DEBUG: 2,
+  INFO: 3,
+  WARNING: 4,
+  ERROR: 5,
+  CRITICAL: 6
+};
+
+const logLevelAsText = [
+  'OFF',
+  'TRACE',
+  'DEBUG',
+  'INFO',
+  'WARNING',
+  'ERROR',
+  'CRITICAL'
+];
+
+function logLevelToString (level) {
+  return logLevelAsText[level] || '???';
```

Comment:
Do you think '???' is the correct text to return here?

Reply:
Not at all but got out of ideas :D Do you have any suggestion?

Reply:
Could return simply undefined?